src/extension/background.js

```diff
@@ -34,4 +34,4 @@ chrome.runtime.onConnect.addListener(function (port) {
     }
     
   });
-});
+});
```

Comment:
Add a line in the end to fix this error

```suggestion
});

```